README.md

```diff
@@ -79,6 +79,7 @@ Coop | Business Areas | Region/Country | Notes
 [Tecso](http://www.tecso.coop/en/) | Development, Information services | Rosario, Argentina | Founded 2002; 150+ associates
 [Vulk Coop](http://vulk.coop) | Design, Development | Austin, TX, USA |
 [Webarchitects Co-operative](https://www.webarchitects.coop) | Hosting, Sysadmin | Sheffield, UK | Multi-stakeholder, including workers, organisations and investors |
+[Coopdevs](http://coopdevs.org/) | Development | Barcelona, Catalonia | Focusing on non-profit & grassroots social change projects e.g. Open Food Network amongst others |
```

Comment:
Maybe we should specify that we're a non profit association and a workers cooperative.

Reply:
Agreed! Fixed. (The non-profit bit is a bit redundant but I added it anyway.)